src/app/components/newsitem/newsitem.component.ts

```diff
@@ -14,13 +14,13 @@ export class NewsitemComponent implements OnInit {
   newsList$: Observable<News[]> = of([]);
   shortNews = true;
   btnName = 'Show full news';
-  a!: string | undefined;
+  abbreviatedNotation!: string | undefined;
```

Comment:
In my opinion the most appropriate names are: shortNews, newsPreview, reducedNewsDescription etc.